### ISSUE TITLE
Exception calling "AddPrivileges" when installing 2016_05_15 release via chocolatey

### ISSUE THREAD

**Issue by alexchandel** [PowerShell/Win32-OpenSSH#232]
Installing with `/SSHServerFeature and /KeyBasedAuthenticationFeature`.

The relevant log extract:

```
2016-05-31 22:52:15,639 [INFO ] -  [SC] ChangeServiceConfig SUCCESS
2016-05-31 22:52:15,831 [INFO ] -  String account: 30
2016-05-31 22:52:15,894 [ERROR] -  Exception calling "AddPrivileges" with "2" argument(s): "The parameter is incorrect"
2016-05-31 22:52:15,898 [ERROR] -  At C:\ProgramData\chocolatey\lib\win32-openssh\tools\chocolateyinstall.ps1:214 char:49
2016-05-31 22:52:15,901 [ERROR] -  +   [MyLsaWrapper.LsaWrapperCaller]::AddPrivileges <<<< ("NT SERVICE\SSHD", "SeServiceLogonRight")
2016-05-31 22:52:15,904 [ERROR] -      + CategoryInfo          : NotSpecified: (:) [], ParentContainsErrorRecordException
2016-05-31 22:52:15,907 [ERROR] -      + FullyQualifiedErrorId : DotNetMethodException
2016-05-31 22:52:15,952 [DEBUG] - Command ['"C:\Windows\System32\WindowsPowerShell\v1.0\powershell.exe" -NoProfile -NoLogo -ExecutionPolicy Bypass -Command "[System.Threading.Thread]::CurrentThread.CurrentCulture = '';[System.Threading.Thread]::CurrentThread.CurrentUICulture = ''; & import-module -name 'C:\ProgramData\chocolatey\helpers\chocolateyInstaller.psm1'; & 'C:\ProgramData\chocolatey\helpers\chocolateyScriptRunner.ps1' -packageScript 'C:\ProgramData\chocolatey\lib\win32-openssh\tools\chocolateyinstall.ps1' -installArguments '' -packageParameters '/SSHServerFeature'"'] exited with '1'
2016-05-31 22:52:15,963 [DEBUG] - Calling command ['"shutdown" /a']
2016-05-31 22:52:15,983 [DEBUG] - Command ['"shutdown" /a'] exited with '1116'
```

Even manually performing the `ntrights.exe` fix from #227 doesn't stop this error.

**Comment by DarwinJS** [comment]
What version of Windows are you installing on?  Just tested again on server 2012 R2 with no problems.  Thanks.

**Comment by alexchandel** [comment]
2008 R2 Standard, 64-bit.

Why are weird Powershell/.NET commands even used when we package ntrights.exe?

**Comment by DarwinJS** [comment]
Because people want Win32-Openssh on nano and NTRIGHTS.exe does not work there: https://github.com/DarwinJS/ChocoPackages/issues/3

**Comment by alexchandel** [comment]
Maybe ntrights should be preferred except on nano.

It's too bad ntrights can't be fixed to support nano.

**Comment by alexchandel** [comment]
Update: I also get this error on Windows 7 Home Premium 64-bit. ntrights is seeming more and more preferable...

**Comment by DarwinJS** [comment]
@alexchandel  - Could you please test if this code works on these cases: https://github.com/proxb/PoshPrivilege/blob/master/PoshPrivilege/Scripts/Add-Privilege.ps1

**Comment by DarwinJS** [comment]
Version 2016.04.15.201601116 is in moderation and uses a new approach for add-privileges if you wish to do early testing.

Might want to use a clean VM to ensure that the right is not already given to the user in question.

**Comment by alexchandel** [comment]
Just tested 2016.04.15.201601116 on the same Win7 machine, seemed to work without issue.

**Comment by alexchandel** [comment]
@DarwinJS Is there a particular reason/known issue why the new package doesn't use the 5_30_2016 release?

**Comment by DarwinJS** [comment]
I must have missed the release announcement - this project is generating more email than it use to.

The new version is in chocolatey moderation.

Thanks for the nudge.

**Comment by DarwinJS** [comment]
Should be fixed in 2016.05.30 version of chocolatey package.  Additional improvements include using ntrights.exe when powershell is older than version 2.0.

In the future please report chocolatey packaging problems here: https://github.com/DarwinJS/ChocoPackages/issues

Thanks.

**Comment by alexchandel** [comment]
Confirmed fixed in the 2016.05.30 version.
